.github/workflows/main.yml

```diff
@@ -65,6 +69,12 @@ jobs:
             laravel: 11.*
           - php: 8.1
             laravel: 11.*
+          - php: 8.2
```

Comment:
why did you excluded this from matrix?

Reply:
laravel 11 still supprots php 8.2-8.4
https://laravel.com/docs/11.x/releases#support-policy

Reply:
My bad, I misunderstood what this was about. I was thinking I was enabling it. LOL